packages/server/src/migrations/0.0.1-init.ts

```diff
@@ -0,0 +1,179 @@
+import { Migration } from '@botpress/messaging-engine'
+
+export class InitMigration extends Migration {
+  meta = {
+    name: InitMigration.name,
+    description: 'Create initial tables',
+    version: '0.0.1'
+  }
+
+  async valid() {
+    return true
+  }
+
+  async applied() {
+    return this.isDown
+  }
+
+  async up() {
+    await this.trx.schema.createTable('msg_meta', (table) => {
+      table.timestamp('time').primary()
+      table.jsonb('data')
+    })
+
+    await this.trx.schema.createTable('msg_kvs', (table) => {
+      table.uuid('id').primary()
+      table.string('key').unique().notNullable()
+      table.jsonb('value').notNullable()
+    })
+
+    await this.trx.schema.createTable('msg_channels', (table) => {
+      table.uuid('id').primary()
+      table.string('name').unique().notNullable()
```

Comment:
We need this massive initial migration to be coherent across the system. That way you can do something like `migrate up -t 0.0.1` on a messaging server that hasn't created its database yet, and this will create a database compatible for version 0.0.1. Same goes for any other version as well. So now our database tests will be able to compare sequential migration runs to known schemas, and individual migration tests will be able to demand the database version they need by migrating to it from version 0.0.1 before doing their tests